src/main/java/duke/tasklist/TaskListDuke.java

```diff
@@ -0,0 +1,139 @@
+package duke.tasklist;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+
+import duke.task.Task;
+
+/**
+ * TaskList stores the list of tasks.
+ *
+ * @author Zhen Xuan (Tutorial Group W12)
+ * @version CS2103T AY21/22 S1
+ */
+public class TaskListDuke implements TaskList {
+    private static final String ADD = "Got it. I've added this task:\n";
+    private static final String LIST_INTRO = "Here are the tasks in your list:\n";
+    private static final String DONE = "Nice! I've marked this task as done:\n";
+    private static final String DELETE = "Noted. I've removed this task:\n";
+    private ArrayList<Task> list;
+
+    /**
+     * Constructor for TaskList.
+     */
+    public TaskListDuke(ArrayList<Task> list) {
+        this.list = list;
+    }
+
+    /**
+     * Adds the task into the list.
+     *
+     * @param task the task to be added into the list
+     */
```

Comment:
I think you should add a parameter description for the returned String (eg @return Current size of the list).